packages/components/core/common-behaviors/color.ts

```diff
@@ -73,6 +73,42 @@ export function mixinColor<T extends Constructor<HasElementRef>>(
             super(...args);
 
             this.color = defaultColor;
+
+            console.warn('mixinColor deprecated and will be deleted in next major release');
```

Comment:
давай добавим проверку, чтобы в проде не вылезло:
```suggestion
            if (isDevMode()) console.warn('mixinColor is deprecated and will be removed in the next major release');
```

<br />

и саму функцию бы пометить как `@deprecated`

Reply:
почему ? пусть наоборот стреляет сразу и где только можно. Вообще сомневаюсь, что кто то юзает это в продуктах